recipes/libcublas/post-link.sh

```diff
@@ -0,0 +1,3 @@
+echo "By downloading and using the ${PKG_NAME} conda package, you accept the terms"  >> $PREFIX/.messages.txt
+echo "and conditions of the NVIDIA cuQuantum Software License Agreement:"            >> $PREFIX/.messages.txt
+echo "https://docs.nvidia.com/cuda/cuquantum/license.html"                           >> $PREFIX/.messages.txt
```

Comment:
@adibbley, this seems to be referencing cuQuantum (even though this is cuBLAS). Is this meant to be here?

Reply:
I haven't added any such files, I think we can remove this.

Reply:
This should be replaced by a pointer to CUDA EULA right?

Reply:
Nope. We are just handling this with `license_url` in `about`

Reply:
Got it, thx

Reply:
Should we keep this file, if yes is there approved/example text for it? If no, is removing it all that needs doing or does something else need adjusting as well?

Reply:
Gotcha thanks for the context

Yeah we should drop it. Nothing else is needed outside removing the file

Reply:
Done